viewer/viewerUtils.js

```diff
@@ -284,15 +284,15 @@ class ViewerUtils {
 
       if (reqQuery.date === '-1' || // An all query
           Config.get('queryAllIndices', Config.get('multiES', false))) { // queryAllIndices (default: multiES)
-        req._arkimeESQueryIndices = Db.fixIndex(['sessions2-*', 'sessions3-*']);
-        return finalCb(err || lerr, query, Db.fixIndex(['sessions2-*', 'sessions3-*'])); // Then we just go against all indices for a slight overhead
+        req._arkimeESQueryIndices = Db.fixIndex(Db.defaultIndexPatterns(Config.getArray('queryExtraIndices', '')));
```

Comment:
Throughout the code the several places where `['sessions2-*', 'sessions3-*']` are passed into db.js have been replaced by `Db.defaultIndexPatterns` which takes an `extraIndices` array (pulled from the config's `queryExtraIndices` value) and merges its contents with the default `['sessions2-*', 'sessions3-*']` array. This merged array is returned.